utils_test.py

```diff
@@ -479,3 +479,9 @@ def test_get_time_in_millisecs_with_complicated_time(self):
         self.assertEqual(
             dt,
             datetime.datetime.fromtimestamp(python_utils.divide(msecs, 1000.0)))
+
+    def test_get_current_appengine_environment(self):
+        os.environ['APPENGINE_RUNTIME'] = 'True'
```

Comment:
Should probably do a self.swap, or find some way to reset to the default thereafter?

Reply:
Done.